Test for `trace_manager.py` is failing

A test is failing for `trace_manager.py` located in `tests/unit/tracing/test_trace_manager.py`.
- Failing method is [test_is_entry_invalid_not_colored](https://github.com/kytos-ng/sdntrace/blob/ca96a370c755b0bd66dbb4f679af926371b7a1b0/tests/unit/tracing/test_trace_manager.py#L95-L102)
- Message: 
```
>       self.assertEqual(entry, "Switch not Colored")
E       AssertionError: <napps.amlight.sdntrace.tracing.trace_ent[38 chars]bb20> != 'Switch not Colored'
```

--- a/tests/unit/tracing/test_trace_manager.py
+++ b/tests/unit/tracing/test_trace_manager.py
@@ -92,8 +92,10 @@ class TestTraceManager(TestCase):
 
         self.assertEqual(entry, "Error: dpid not provided")
 
-    def test_is_entry_invalid_not_colored(self):
+    @patch("napps.amlight.sdntrace.shared.colors.Colors.get_switch_color")
+    def test_is_entry_invalid_not_colored(self, mock_colors):
         """Test if the entry request does not have a valid color."""
+        mock_colors.return_value = {}
         eth = {"dl_vlan": 100}
         dpid = {"dpid": "00:00:00:00:00:00:00:01", "in_port": 1}
         switch = {"switch": dpid, "eth": eth}

